entry/src/block_component.rs

```diff
@@ -944,14 +955,18 @@ impl BlockFooterV1 {
             return Err(BlockComponentError::InsufficientData);
         }
 
-        // Read timestamp
-        // Unwrap: HEADER_SIZE = TIMESTAMP_SIZE + USER_AGENT_LEN_SIZE > TIMESTAMP_SIZE, so this will
-        // never fail.
-        let time_bytes = data[..Self::TIMESTAMP_SIZE].try_into().unwrap();
+        // Read bank hash; HEADER_SIZE > HASH_SIZE, so this unwrap will never fail.
+        let hash_bytes: [u8; 32] = data[..Self::HASH_SIZE].try_into().unwrap();
```

Comment:
nit: I personally prefer to forget the knowledge `HEADER_SIZE > HASH_SIZE `and just do
`.map_err(|_| BlockComponentError::InsufficientData)?;`
everywhere, be dumb so code will not fail when other engineers come in and change it.